services/cd-service/pkg/repository/transformer_db_test.go

```diff
@@ -2701,13 +2810,35 @@ func TestDeleteEnvFromAppWithDB(t *testing.T) {
 				if err2 != nil {
 					return fmt.Errorf("error retrieving release: %v", err2)
 				}
-				for _, release := range releases {
-					for env, manifest := range tc.ExpectedManifests {
-						if diff := cmp.Diff(manifest, release.Manifests.Manifests[env]); diff != "" {
-							return fmt.Errorf("error mismatch Manifests - want, +got:\n%s", diff)
+
+				if tc.ExpectedManifestsByEachRelease != nil {
+					for releaseNumber, expectedManifests := range tc.ExpectedManifestsByEachRelease {
+						var release *db.DBReleaseWithMetaData
+						for _, rel := range releases {
+							if *rel.ReleaseNumbers.Version == releaseNumber {
+								release = rel
+								break
+							}
+						}
+						if release == nil {
+							return fmt.Errorf("expect release but not found: %v", err2)
```

Comment:
err2 is always nil here, please add something else to the returned error, like  the app name